apps/github-stack/src/github-ineraction/domain/scheduler/github-scheduler.ts

```diff
@@ -44,18 +48,25 @@ export class GitHubScheduler {
       console.log(
         `For repository ${gitRepository.full_name} latest reliase is not found`
       );
+      console.error(error);
     }
   }
 
   async checkForUpdates() {
     const repositories = await this.gitRepository.find({ relations: ["user"] });
     for (const repo of repositories) {
-      const release = await this.getLatestReliase(repo);
-      if (repo.latestRelease != release) {
-        repo.latestRelease = release;
-        this.gitRepository.save(repo);
-        await this.sendingNotification(repo);
-      }
+      const latestRelease = await this.getLatestReliase(repo);
+      repo.releases?.forEach((release) => {
+        if (release.release != latestRelease) {
+          this.releaseRep.create({
+            release: latestRelease,
+            release_date: new Date(),
+            repository: repo,
+          });
+          this.gitRepository.save(repo.releases);
+          this.sendingNotification(repo);
+        }
+      });
     }
   }
   async sendingNotification(repo: GitRepository) {
```

Comment:
sendingNotification should be renamed to sendNotification. Naming should suggest action that will be taken